app/Commands/Servers/DeleteCommand.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Commands\Servers;
+
+use App\Commands\BaseCommand;
+
+class DeleteCommand extends BaseCommand
+{
+    protected $signature = 'servers:delete {id?} {--force} {--profile=} {--delete-on-provider}';
+
+    protected $description = 'Delete a server';
+
+    protected $simpleOutput = true;
+
+    public function action(): int
+    {
+        $serverId = $this->argument('id');
+
+        if (empty($serverId)) {
+            $serverId = $this->askToSelectServer('Which server would you like to delete?');
+        }
+
+        $delete = $this->option('force') || $this->confirm('Are you sure you want to delete this server?');
```

Comment:
Before asking them to confirm, let's show an alert:

> You're about to delete "SERVER NAME"

Then update the confirm wording to:

> Do you wish to continue?